python/daqconf/core/conf_utils.py

```diff
@@ -491,36 +510,29 @@ def generate_boot(apps: list, ers_settings=None, info_svc_uri="file://info_${APP
         }
 
     daq_app_specs = {
-        "daq_application_ups" : {
-            "comment": "Application profile based on a full dbt runtime environment",
-            "env": {
-                "DBT_AREA_ROOT": "getenv",
-                "TRACE_FILE": "getenv:/tmp/trace_buffer_${HOSTNAME}_${USER}",
-            },
-            "cmd": ["CMD_FAC=rest://localhost:${APP_PORT}",
-                    "INFO_SVC=" + info_svc_uri,
-                    "cd ${DBT_AREA_ROOT}",
-                    "source dbt-env.sh",
-                    "dbt-workarea-env",
-                    "cd ${APP_WD}",
-                    "daq_application --name ${APP_NAME} -c ${CMD_FAC} -i ${INFO_SVC}"]
-        },
         "daq_application" : {
-            "comment": "Application profile using  PATH variables (lower start time)",
+            "comment": "Application profile using PATH variables (lower start time)",
+            "image": image,
             "env":{
                 "CET_PLUGIN_PATH": "getenv",
                 "DETCHANNELMAPS_SHARE": "getenv",
                 "DUNEDAQ_SHARE_PATH": "getenv",
                 "TIMING_SHARE": "getenv",
                 "LD_LIBRARY_PATH": "getenv",
                 "PATH": "getenv",
-                # "READOUT_SHARE": "getenv",
-                "TRACE_FILE": "getenv:/tmp/trace_buffer_${HOSTNAME}_${USER}",
+                "TRACE_FILE": "getenv:/tmp/trace_buffer_{APP_HOST}_{DUNEDAQ_PARTITION}",
+                "CMD_FAC": "rest://localhost:{APP_PORT}",
+                "INFO_SVC": info_svc_uri,
             },
-            "cmd": ["CMD_FAC=rest://localhost:${APP_PORT}",
-                    "INFO_SVC=" + info_svc_uri,
-                    "cd ${APP_WD}",
-                    "daq_application --name ${APP_NAME} -c ${CMD_FAC} -i ${INFO_SVC}"]
+            "cmd":"daq_application",
+            "args": [
+                "--name",
+                "{APP_NAME}",
+                "-c",
+                "{CMD_FAC}",
+                "-i",
+                "{INFO_SVC}"
+            ]
         }
     }
 
```

Comment:
I'm puzzled: control port numbers are supposed to be fixed in kubernetes and staggered when running over ssh. But there is no distinction here?

Reply:
Indeed, I don't think there is any reason why it shouldn't be staggered for k8s too.

Reply:
Because it's not how kubernetes works. With each application encapsulated in a container, identified by a different DNS name, all command (and other service) ports must be the same.

Reply:
Yeah, indeed, and in fact this parameter is completely ignored by k8spm.py https://github.com/DUNE-DAQ/nanorc/blob/plasorak/k8s-np04-cluster/src/nanorc/k8spm.py#L549
So the prefered way to fix this would be to create a new `daq_application_k8s` section in `boot.json` were the k8s information is separated?

Reply:
Actually sorry this is more relevant for the `image` comment. For the commands ports, I don't know how to sort it without differentiating between k8s/non-k8s configurations, which I thought was something we wanted to avoid.

Reply:
Right, keeping the same configuration is desirable but at a reasonable cost. Clearly we're in a situation where we don't have a way to keep a single consistent configuration w/o injecting confusing/contradictory information.
Regarding `k8spm` ignoring the command port, it is a hack from the prototyping days! Silently overriding configuration parameters is very likely to lead to errors/confusion and must be avoided (for real). 
So I don't see a way not to differentiate between ssh and kubernetes in daqconf, at this stage